src/components/MainView/ListOfDashboards/DashboardPreview/DashboardPreview.jsx

```diff
@@ -1,6 +1,7 @@
 import React, { Component } from 'react';
 import { Button, Card } from 'react-bootstrap';
 import { Link } from 'react-router-dom';
+import { db } from '../../../../fire';
```

Comment:
db is exported by default. Then it should be imported accordingly: `import db from '../../../../fire';`